src/config.cpp

```diff
@@ -35,13 +35,7 @@ Config::Config(bool has_cmd)
     { ConfigKeyString::license, { .value = std::string("GPL") } },
     { ConfigKeyString::str_trunc_trailer, { .value = std::string("..") } },
     { ConfigKeySymbolSource::default_,
-      { .value =
-#ifdef HAVE_LIBLLDB
-            ConfigSymbolSource::dwarf
-#else
-            ConfigSymbolSource::symbol_table
-#endif
-      } },
+      { .value = ConfigSymbolSource::symbol_table } },
```

Comment:
I think this entire config can go and perhaps we should issue a warning if we see it but not break the script if it's set to "symbol_table"